pkgs/tools/misc/plantuml/plantuml-c4.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, stdenv, makeWrapper, fetchzip, runCommand, plantuml, plantuml-c4 }:
+
+# The C4-PlantUML docs say that it suffices to run plantuml with the
+# -DRELATIVE_INCLUDE="..." arg to make plantuml find the C4 templates
+# when included like "!include C4_Container.puml".
+# Unfortunately, this is not sufficient in practise.
+# What helps is setting the -Dplantuml.include.path="..." *before* the jar
+# parameter.
+# The -DRELATIVE_INCLUDE param then *still* needs to be set, because the
+# C4 template vars check for existence of this variable and if it is not set,
+# reference paths in the internet.
+
+let
+
+in
+
+stdenv.mkDerivation rec {
+  pname = "plantuml-c4";
+  version = "2022.08.21";
```

Comment:
Please format the date as yyyy-mm-dd. (And I wonder if the version attr should be "unstable-yyyy-mm-dd"?)

Reply:
ok, done.

Reply:
Commit message still uses the non-date format.

Reply:
fixed